Cyan/types.py

```diff
@@ -350,7 +355,7 @@ def is_truthy(self) -> Bool:
         return Bool(self.value)
 
     # converters
-    def to_Number(self) -> RTResult:
+    def to_number(self) -> RTResult:
```

Comment:
That isn't going to work. the to_{type} method names has to be exactly same as the real type name, it has to be to_Number and not to_number

Reply:
there's multiple of these `to_type` methods do they also need to be changed?
or is it only this one

Reply:
They all need to be the exact as the type name, but I don't see any other `to_type` method renamed